src/components/EventForm/EventForm.js

```diff
@@ -65,7 +65,6 @@ function EventForm({ courses, friends, hostId }) {
       hostId,
       selectedFriends
     )
-      .then((json) => console.log(json))
       .catch((e) => console.log(e));
```

Comment:
Good catch here.  Let's make a note to display the error message if it ends up in the catch instead of console logging